packages/tools/accessibility/src/accessibilityNodeItem.ts

```diff
@@ -0,0 +1,137 @@
+    }
+
+    /**
+     * If this entity is focusable (can be focused by tab key pressing).
+     */
+    public override get isFocusable(): boolean {
+        return true;
+        // if (this._isFocusable) {
+        //     return this._isFocusable;
+        // }
+        // return this.isActionable;
+    }
+
+    /**
+     * Callback when the HTML element is focused. Show visual indication on BabylonJS entity.
+     */
+    public override focus(): void {
+        // If defined eventHandler, override default.
+        let eventHandler = (this.entity as Node).accessibilityTag?.eventHandler;
+        if (eventHandler?.onfocus) {
+            eventHandler.onfocus();
+            return;
+        }
+
+        if (this.entity instanceof Mesh) {
+            const mesh = this.entity as Mesh;
+            mesh.enableEdgesRendering(0.999);
+            mesh.edgesWidth = 5;
+            mesh.edgesColor = new Color4(0.25, 0.5, 1, 1);
+        }
+    }
+
+    /**
+     * Callback when the HTML element is blured. Dismiss visual indication on BabylonJS entity.
+     */
+    public override blur(): void {
+        // If defined eventHandler, override default.
+        let eventHandler = (this.entity as Node).accessibilityTag?.eventHandler;
+        if (eventHandler?.onblur) {
+            eventHandler.onblur();
+            return;
+        }
+
+        if (this.entity instanceof Mesh) {
+            const mesh = this.entity as Mesh;
+            mesh.disableEdgesRendering();
+        }
+    }
+
+    /**
+     * Callback when the HTML element is clicked. Apply that to BabylonJs entity.
+     */
+    public override click(): void {
+        // If defined eventHandler, override default.
+        let eventHandler = (this.entity as Node).accessibilityTag?.eventHandler;
+        if (eventHandler?.onclick) {
+            eventHandler.onclick();
+            return;
+        }
+
+        let actions: IAction[] = [];
+        actions.push(...this._getTriggerActions(this.entity, Constants.ACTION_OnLeftPickTrigger));
+        actions.push(...this._getTriggerActions(this.entity, Constants.ACTION_OnPickTrigger));
+
+        actions.forEach((action) => {
+            action._executeCurrent();
+        });
+    }
+
+    /**
+     * Callback when the HTML element is right clicked. Apply that to BabylonJs entity.
+     */
+    public override rightClick(): void {
+        // If defined eventHandler, override default.
+        let eventHandler = (this.entity as Node).accessibilityTag?.eventHandler;
+        if (eventHandler?.oncontextmenu) {
+            eventHandler.oncontextmenu();
+            return;
+        }
+
+        let actions: IAction[] = [];
+        actions.push(...this._getTriggerActions(this.entity, Constants.ACTION_OnRightPickTrigger));
+        actions.push(...this._getTriggerActions(this.entity, Constants.ACTION_OnPickTrigger));
+
+        actions.forEach((action) => {
+            action._executeCurrent();
+        });
+    }

```

Comment:
Feels like this is repeating code that can be moved to a single private function with different parameters.